src/wh_cryptocb.c

```diff
@@ -400,12 +82,15 @@ int wolfHSM_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
             }
```

Comment:
Still need to remove the `sleep()`s from this function. Line 398 for example (sorry, GH won't let me comment on that line in this review since it wasn't changed)